frontend/.eslintrc.js

```diff
@@ -153,9 +159,9 @@ module.exports = {
        */
       rules: {
         // Use of "any" type disallowed. Use the following if you really need "any": // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        "@typescript-eslint/no-explicit-any": "error",
+        "@typescript-eslint/no-explicit-any": "off",
```

Comment:
@ASpiteri-BCGov Can you provide some context as to why a lot of the linter's rules were softened? Was there a decision reached on the team that resulted in this? Can we document that somewhere?